app/src/main/java/org/oppia/app/utility/Logger.kt

```diff
@@ -0,0 +1,122 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.util.Log
+import org.oppia.app.application.ApplicationContext
+import org.oppia.app.application.OppiaApplication
+
+import java.io.BufferedWriter
+import java.io.File
+import java.io.FileOutputStream
+import java.io.FileWriter
+import java.io.IOException
+import java.io.PrintStream
+import java.text.SimpleDateFormat
+import java.util.Calendar
+import java.util.Locale
+import javax.inject.Inject
+import kotlin.coroutines.coroutineContext
+import android.widget.Toast
+
+/** Wrapper class for Android Log. */
```

Comment:
We should also mention that this performs file logging.

Reply:
ok